Source/DafnyCore/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -1899,9 +1900,6 @@ public ExpressionTranslator(ExpressionTranslator etran, string modifiesFrame)
         if (s is BinaryExpr && aggressive) {
           BinaryExpr bin = (BinaryExpr)s;
           switch (bin.ResolvedOp) {
-            case BinaryExpr.ResolvedOpcode.Union:
-            case BinaryExpr.ResolvedOpcode.Intersection:
-            case BinaryExpr.ResolvedOpcode.SetDifference:
             case BinaryExpr.ResolvedOpcode.MultiSetUnion:
             case BinaryExpr.ResolvedOpcode.MultiSetIntersection:
```

Comment:
I think it would be good to do the analogous change also for multisets. You could do that in this same PR, in a separate PR, or, if nothing else, file an Issue that reminds us that this would be a good change to make in the future.

Reply:
I'll do this in this PR if it's not too hard

Reply:
> I think it would be good to do the analogous change also for multisets

Done ! That wasn't too hard. The only difference was that the trigger expressions were of the form (A + B)[x] and so this is what I split into two new triggers A[x] and B[x] (if the bound variables are the same of course!)